core/src/main/java/io/questdb/griffin/engine/functions/groupby/CountDoubleGroupByFunctionFactory.java

```diff
@@ -53,15 +52,6 @@ public Function newInstance(
             CairoConfiguration configuration,
             SqlExecutionContext sqlExecutionContext
     ) throws SqlException {
-        final Function arg = args.getQuick(0);
```

Comment:
`count(constant)` is erased in the optimiser, it is unnecessary to handle constant in "count" functions anymore

Reply:
i think i did put comment into one, there are a lot of factories!